### PR TITLE
DOC: How to install pypi from any branch

### PR DESCRIPTION
**Pull request by pubpub-zz**
addressed in #2075

### CODE REVIEW

**Comment by pubpub-zz** [comment]
@MartinThoma 
all yours

File: docs/dev/testing.md [hunk 1]
```diff
@@ -58,3 +58,20 @@ If the test is a regression test, write
 
 If the regression test is just one parameter of other tests, then add it as
 a comment for that parameter.
+
+## Evaluate a PR in-progress version
+
+You may want to test a version from a PR which has not been released yet.
+the easiest way is to use pip and install a version from git:
```
Comment (MartinThoma) [inline docs/dev/testing.md:65]:
```suggestion
The easiest way is to use pip and install a version from git:
```

File: docs/dev/testing.md [hunk 2]
```diff
@@ -58,3 +58,20 @@ If the test is a regression test, write
...
+
+a) go the PR and identify the repo and branch.
```
Comment (MartinThoma) [inline docs/dev/testing.md:67]:
```suggestion
a) Go the PR and identify the repository and branch.
```

File: docs/dev/testing.md [hunk 3]
```diff
@@ -58,3 +58,20 @@ If the test is a regression test, write
...
+
+Example from below : repo: __pubpub-zz__ / branch: __iss2200__ :
```
Comment (MartinThoma) [inline docs/dev/testing.md:69]:
```suggestion
Example from below : repository: __pubpub-zz__ / branch: __iss2200__ :
```

File: docs/dev/testing.md [hunk 4]
```diff
@@ -58,3 +58,20 @@ If the test is a regression test, write
...
+![PR Header example](PR_Header_example.png)
+
+b) you can then install the version using pip from git:
+
+example:
```
Comment (MartinThoma) [inline docs/dev/testing.md:74]:
```suggestion
Example:
```

File: docs/dev/testing.md [hunk 5]
```diff
@@ -58,3 +58,20 @@ If the test is a regression test, write
...
+```
+pip install pip install git+https://github.com/pubpub-zz/pypdf.git@iss2200
```
Comment (MartinThoma) [inline docs/dev/testing.md:76]:
```suggestion
pip install git+https://github.com/pubpub-zz/pypdf.git@iss2200
```

**Comment by MartinThoma** [comment]
Nice! Thank you :pray:

**Pull request merged**